cmd/kube-controller-manager/app/plugins.go

```diff
@@ -116,13 +105,6 @@ func ProbeControllerVolumePlugins(logger klog.Logger, config persistentvolumecon
 		klog.FlushAndExit(klog.ExitFlushTimeout, 1)
 	}
 	allPlugins = append(allPlugins, nfs.ProbeVolumePlugins(nfsConfig)...)
-
-	var err error
-	allPlugins, err = appendExpandableLegacyProviderVolumes(logger, allPlugins, utilfeature.DefaultFeatureGate)
```

Comment:
The portworx volume plugin doesn't complete csi migration plan, so we could not remove it from the expand-controller and pv-binder-controller in this release.

According to the previous sig-meeting, its migration plan is not clear. It lacks active maintainers and contributors to push the migration forward. So I don't know if we should remove it directly in this release without a deprecated period. What do you think about it? @xing-yang @jsafrane

Reply:
Regarding Portworx CSI Migration, @lpabon is working on enabling it by default while staying in Beta in 1.31.

Reply:
https://github.com/kubernetes/enhancements/pull/4467

Reply:
@xing-yang  The KEP was merged on Feb 2. but @lpabon commented that [the plan is to remove the in-tree driver for Portworx for 1.31](https://github.com/kubernetes/kubernetes/issues/99518)

Reply:
Correct.  Please note that the milestone is 1.31 in the KEP.  @lpabon initially tried to do it in 1.30, but decided later 1.31 will be the target release.

Reply:
from what i understood ... sounds like nothing changes in this PR.